homeassistant/components/modbus/climate.py

```diff
@@ -105,6 +107,13 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         target_temperature = (
             float(kwargs[ATTR_TEMPERATURE]) - self._offset
         ) / self._scale
+        if ATTR_FORCE in kwargs:
```

Comment:
This parameter isn't allowed in the service schema for set temperature:
https://github.com/home-assistant/core/blob/ea2b5a80db62744d661e26fbf0d1c527a4c371a7/homeassistant/components/climate/__init__.py#L92-L104

Reply:
Good catch, I will make it in another way.

Reply:
Changed including the "breaking change" in the description.